cdc/owner/ddl_manager.go

```diff
@@ -184,6 +198,18 @@ func (m *ddlManager) tick(
 	checkpointTs model.Ts,
 	tableCheckpoint map[model.TableName]model.Ts,
 ) ([]model.TableID, *schedulepb.BarrierWithMinTs, error) {
+	// needBootstrap is true when the the downstream is kafka
+	// and the protocol is simple protocol.
+	if m.needBootstrap {
+		ok, err := m.checkAndBootstrap(ctx)
```

Comment:
It looks that the `checkAndBootstrap` is called on every tick if the `needBootstrap` is true.

can we simplify this?

Reply:
After bootstrap complete, the `checkAndBootstrap ` will return immediately when it is called.